### ISSUE TITLE
Style transfer notebook removed / not functional

### ISSUE THREAD

**Issue by ioannist** [EugenHotaj/pytorch-generative#13]
Style transfer notebook won't open in colab

```
Notebook not found
There was an error loading this notebook. Ensure that the file is accessible and try again.
Ensure that you have permission to view this notebook in GitHub and authorize Colaboratory to use the GitHub API.
```

Also, manually loading the notebook locally or on colab fails in the secondcell

```
---------------------------------------------------------------------------
ImportError                               Traceback (most recent call last)
<ipython-input-15-7857ada21f3b> in <module>
      2 
      3 import kornia
----> 4 from nn_hallucinations import colab_utils
      5 import torch
      6 from torch import nn

ImportError: cannot import name 'colab_utils' from 'nn_hallucinations' (unknown location)
```

**Comment by EugenHotaj** [comment]
Hi @ioannist, thanks for catching this. This library used to be called nn_hallucinations and was renamed to pytorch-generative, which is why you're getting the error. I fixed this is 7e00193f0f59984dfa5f144cda64fe08a7f6a813, please try it again and let me know if you run into any more issues. 

We really need some kind of tests for these notebooks, as they're very prone to getting out of sync and breaking.

**Comment by EugenHotaj** [comment]
Going to close this out as I think it's now fixed. Please reopen if you see the issue again.
